.changelog/unreleased/bug-fixes/ibc/1706-fix-formatting-for-some-tendermint-errors.md

```diff
@@ -1,3 +1,3 @@
 - Fixed the formatting of NotEnoughTimeElapsed and NotEnoughBlocksElapsed
-  in tendermint errors ([#1706](https://github.com/informalsystems/ibc-
```

Comment:
This is provoked from `unclog` (ref https://github.com/informalsystems/unclog/issues/27), thanks for the fix Mikhail!